plugins/zipkin/wiring.go

```diff
@@ -1,9 +1,17 @@
 // Package zipkin provides a plugin to generate and include a zipkin collector instance in a Blueprint application.
 //
-// The package provides a zipkin container that provides the server-side implementation
-// and a go-client for connecting to the server.
+// # Artifacts Generated
 //
-// The applications must use a backend.Tracer (runtime/core/backend) as the interface in the workflow.
+//  1. The package provides a zipkin container that provides the server-side implementation of the Jaeger collector.
+//  2. Generates a go-client for connecting to the zipkin server. The client is set as the default exporter for the opentelemetry tracer which is then used by the opentelemetry plugin to correctly export spans to the Jaeger collector.
```

Comment:
The plugin doesn't generate a go client; it uses the client from the [runtime/plugins/zipkin] package. Can you change this to describe that better.